2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0639\u0631\u0628001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0101','\u0639\u0631\u0628\u064a\u0629','Arabe','01','\u0639\u0631\u0628001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0625\u0639\u0631001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0102','\u0625\u0646\u0634\u0627\u0621','R\u00e9daction arabe','01','\u0625\u0639\u0631001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0644\u0639\u0631002' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0103','\u0644\u063a\u0640\u0640\u0629 \u0639\u0631\u0628\u064a\u0629','Langue arabe','01','\u0644\u0639\u0631002','','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0631\u062c003' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0104','\u062a\u0639\u0631\u064a\u0628','Traduction','01','\u062a\u0631\u062c003','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0628\u0644\u0627002' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0105','\u0628\u0644\u0627\u063a\u0629 \u0639\u0631\u0628\u064a\u0629','R\u00e9thorie','01','\u0628\u0644\u0627002','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0645\u0639\u0631003' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0106',' \u0639\u0631\u0628\u064a\u0640\u0629:\u0646\u0634\u0627\u0637 \u0634\u0641\u0648\u064a','Arabe (Activit\u00e9 Orale)','01','\u0645\u0639\u0631003','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0641\u0631\u0646001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0201','\u0641\u0631\u0646\u0633\u064a\u0629','Fran\u00e7ais','02','\u0641\u0631\u0646001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0627\u0646\u0642001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0301','\u0623\u0646\u0642\u0644\u064a\u0632\u064a\u0629','Anglais','03','\u0627\u0646\u0642001','','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0641\u0644\u0633001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0401','\u0641\u0644\u0633\u0641\u0629','Philosophie','04','\u0641\u0644\u0633001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0645\u062f\u062e001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0402','\u0645\u062f\u062e\u0644 \u0625\u0644\u0649 \u0627\u0644\u0641\u0644\u0633\u0641\u0629','Initiation philosophique','04','\u0645\u062f\u062e001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0627\u0631001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0501','\u062a\u0627\u0631\u064a\u062e','Histoire','05','\u062a\u0627\u0631001','07','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062c\u063a\u0631001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0502','\u062c\u063a\u0631\u0627\u0641\u064a\u0627','Geographie','05','\u062c\u063a\u0631001','06','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0648\u062c001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0503','\u062a\u0627\u0631\u064a\u062e \u0648 \u062c\u063a\u0631\u0627\u0641\u064a\u0627','Histoire et G\u00e9ograph','05','\u062a\u0648\u062c001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0625\u0633001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0601','\u062a\u0631\u0628\u064a\u0629 \u0625\u0633\u0644\u0627\u0645\u064a\u0629','Education Islamique','06','\u062a\u0625\u0633001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0641\u0643001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0602','\u062a\u0641\u0643\u064a\u0631 \u0625\u0633\u0644\u0627\u0645\u064a','Pens\u00e9es Islamique','06','\u062a\u0641\u0643001','10','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0648\u0637001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0701','\u062a\u0631\u0628\u064a\u0629 \u0645\u062f\u0646\u064a\u0629','Education Civique','07','\u062a\u0648\u0637001','','2','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0631\u064a\u0627001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0801','\u0631\u064a\u0627\u0636\u064a\u0627\u062a','Math\u00e9matiques','08','\u0631\u064a\u0627001','12','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0639\u0641\u0649001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('0901','\u0639\u0644\u0648\u0645 \u0641\u064a\u0632\u064a\u0627\u0626\u064a\u0629','Sciences physiques','09','\u0639\u0641\u0649001','15','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0639\u0637\u0628001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1001','\u0639\u0644\u0648\u0645 \u0628\u064a\u0648\u0644\u0648\u062c\u064a\u0629','Sciences biologique','10','\u0639\u0637\u0628001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u062a\u0642001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1101','\u062a\u0631\u0628\u064a\u0629 \u062a\u0643\u0646\u0648\u0644\u0648\u062c\u064a\u0629','Education Technologique','11','\u062a\u062a\u0642001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0627\u0642\u062a001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1201','\u0625\u0642\u062a\u0635\u0627\u062f','Economie','12','\u0627\u0642\u062a001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0635\u0631001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1301','\u062a\u0635\u0631\u0641','Gestion','13','\u062a\u0635\u0631001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0642\u0646001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1415','\u0645\u0648\u0627\u062f \u062a\u0642\u0646\u064a\u0629','Matieres Techniques','14','\u062a\u0642\u0646001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u062a\u0634001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1601','\u062a\u0631\u0628\u064a\u0629 \u062a\u0634\u0643\u064a\u0644\u064a\u0629','Education Artistique','16','\u062a\u062a\u0634001','16','4','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0623\u0644\u0645001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1701','\u0623\u0644\u0645\u0627\u0646\u064a\u0629','Allemand','17','\u0623\u0644\u0645001','20','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0625\u064a\u0637001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1801','\u0625\u064a\u0637\u0627\u0644\u064a\u0629','Italien','18','\u0625\u064a\u0637001','19','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0625\u0633\u0628001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('1901','\u0625\u0633\u0628\u0627\u0646\u064a\u0629','Espagnol','19','\u0625\u0633\u0628001','21','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0625\u0639\u0644001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('2001','\u0625\u0639\u0644\u0627\u0645\u064a\u0629','Informatique','20','\u0625\u0639\u0644001','11','5','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0628\u062f001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('2101','\u062a\u0631\u0628\u064a\u0629 \u0628\u062f\u0646\u064a\u0629','Education physique','21','\u062a\u0628\u062f001','','1','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0645\u0648001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('2201','\u062a\u0631\u0628\u064a\u0629 \u0645\u0648\u0633\u064a\u0642\u064a\u0629','Education musicale','22','\u062a\u0645\u0648001','17','4','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u0644\u062d\u064a001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('2401','\u0644\u063a\u0629 \u062d\u064a\u0629 \u062b\u0627\u0644\u062b\u0629','Troisi\u00e8me langue vivante','24','\u0644\u062d\u064a001','','3','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CODEMATISY VARCHAR(20)": "'\u062a\u0639\u0625001' (21)"; SQL statement:
INSERT INTO MATIERE VALUES('3501','\u062a\u0639\u0644\u0651\u0645\u0627\u062a \u0627\u062e\u062a\u064a\u0627\u0631\u064a\u0651\u0629','Education Optionnel','','\u062a\u0639\u0625001','','4','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0623.\u0645.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('10','\u0623\u0646\u0634\u0637\u0629 \u0645\u062e\u062a\u0644\u0641\u0629','Activit\u00e9s Diverses','\u0623.\u0645.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0634\u0641.' (13)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('11','\u0634\u0641\u0627\u0647\u064a','Oral','\u0634\u0641.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0623.\u062a.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('12','\u0623\u0634\u063a\u0627\u0644 \u062a\u0637\u0628\u064a\u0642\u064a\u0629','T.P','\u0623.\u062a.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0643\u0631\u0627\u0633' (24)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('13','\u0639\u062f\u062f \u0644\u0644\u0643\u0631\u0651\u0627\u0633','Note pour le cahier','\u0643\u0631\u0627\u0633','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0645\u0644.\u0645' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('14','\u0645\u0644\u0627\u062d\u0638\u0629 \u0645\u0633\u062a\u0645\u0631\u0629','Observation continue','\u0645\u0644.\u0645','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0645\u0639.\u0634' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('15','\u0645\u0639\u062f\u0644 \u0627\u0644\u0634\u0641\u0648\u064a','Moyenne des oraux','\u0645\u0639.\u0634','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0639.\u0645\u0634' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('16','\u0639\u062f\u062f \u0645\u0634\u0631\u0648\u0639 ','Project Work','\u0639.\u0645\u0634','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u062b\u0642.\u0631' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('17','\u062b\u0642\u0627\u0641\u0629 \u0631\u064a\u0627\u0636\u064a\u0629 \u0648 \u0627\u0644\u0631\u0648\u062d \u0627\u0644\u0623\u0648\u0644\u0645\u0628\u064a\u0629','Culture et esprie sportive','\u062b\u0642.\u0631','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0645\u0646\u062a\u0648' (24)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('18','\u0639\u062f\u062f \u0645\u0646\u062a\u0648\u062c','Note productivite','\u0645\u0646\u062a\u0648','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0645\u062a\u0627\u0628' (24)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('19','\u0639\u062f\u062f \u0645\u062a\u0627\u0628\u0639\u0629','Note Suivi','\u0645\u062a\u0627\u0628','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0627.\u0643.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('20','\u0625\u062e\u062a\u0628\u0627\u0631 \u0643\u062a\u0627\u0628\u064a','test ','\u0627.\u0643.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0646.\u062e\u0635' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('21','\u0627\u062e\u062a\u0628\u0627\u0631 \u0646\u0638\u0631\u064a \u062e\u0635\u0648\u0635\u064a','Culture et esprie sportive','\u0646.\u062e\u0635','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0627.\u062a\u0634' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('23','\u0627\u062e\u062a\u0628\u0627\u0631 \u062a\u0634\u0643\u064a\u0644\u064a','Test Artistique','\u0627.\u062a\u0634','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0627\u0633.\u0643' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('24','\u0627\u062e\u062a\u0628\u0627\u0631 \u0633\u0645\u0639\u064a \u0643\u062a\u0627\u0628\u064a','Test Auditif ecrit','\u0627\u0633.\u0643','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0641.\u0645.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('30','\u0641\u0631\u0636 \u0645\u0640\u0640\u0631\u0627\u0642\u0628\u0629','devoir de contr\u00f4le','\u0641.\u0645.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0639.\u062a.\u062f' (20)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('32','\u0639\u062f\u062f \u0627\u0644\u062a\u0642\u064a\u064a\u0645 \u0627\u0644\u062f\u0648\u0631\u064a','Note d evaluation Periodique','\u0639.\u062a.\u062f','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0641.\u062a.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('40','\u0641\u0631\u0636 \u062a\u0623\u0644\u064a\u0641\u064a','Devoir de Synthese','\u0641.\u062a.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0627.\u062a\u0637' (19)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('41','\u0625.\u062a\u0637\u0628\u064a\u0642\u064a ','Test Th.Collect.','\u0627.\u062a\u0637','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:08:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "ABRETYPEEPREAR VARCHAR(10)": "'\u0627.\u0648.' (14)"; SQL statement:
INSERT INTO TYPEEPRE VALUES('50','\u0627\u0645\u062a\u062d\u0627\u0646 \u0648\u0637\u0646\u064a(\u0646\u0647\u0627\u064a\u0629 \u0645\u0631\u062d\u0644\u0629)','Examen national (fin cycle)','\u0627.\u0648.','2020-11-18 00:00:00.000000') [22001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE DISCIPLINE[*]"; expected "TABLE"; SQL statement:
TRUNCATE DISCIPLINE [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE EPREUVE[*]"; expected "TABLE"; SQL statement:
TRUNCATE EPREUVE [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE MATIERE[*]"; expected "TABLE"; SQL statement:
TRUNCATE MATIERE [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE NIVESCOL[*]"; expected "TABLE"; SQL statement:
TRUNCATE NIVESCOL [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE NOTEELEV[*]"; expected "TABLE"; SQL statement:
TRUNCATE NOTEELEV [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE PROGENSE[*]"; expected "TABLE"; SQL statement:
TRUNCATE PROGENSE [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE TYPEEPRE[*]"; expected "TABLE"; SQL statement:
TRUNCATE TYPEEPRE [42001-200]
2020-12-29 16:15:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "TRUNCATE TYPEMATI[*]"; expected "TABLE"; SQL statement:
TRUNCATE TYPEMATI [42001-200]
2020-12-29 16:49:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODEAMAT" not found; SQL statement:
SELECT * FROM EPREUVE WHERE codeamat = '2001' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 16:49:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODEAMATI" not found; SQL statement:
SELECT * FROM EPREUVE WHERE codeamati = '2001' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 19:46:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODETYPEEPRE"; SQL statement:
SELECT LIBETYPEEPREAR, COEFEPRE, ABRETYPEEPREAR, CODETYPEEPRE
FROM EPREUVE AS E INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
WHERE CODEMATI = '2001' AND CODENIVE='312414' AND CODEPERIEXAM = '31' [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.command.dml.Select.mapColumns(Select.java:1600)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 19:59:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT CODEMATI, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM 
     EPREUVE AS E 
        INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
        INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
WHERE CODEMATI = '2001' AND CODENIVE='312414' AND CODEPERIEXAM = '31' [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.command.dml.Select.mapColumns(Select.java:1600)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 19:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT E.CODEMATI, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM 
     EPREUVE AS E 
        INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
        INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
WHERE CODEMATI = '2001' AND CODENIVE='312414' AND CODEPERIEXAM = '31' [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.expression.condition.Comparison.mapColumns(Comparison.java:505)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1603)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 20:00:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT E.CODEMATI, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM 
     EPREUVE AS E 
        INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
        INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
WHERE CODEMATI = '2001' AND CODENIVE='312414' AND CODEPERIEXAM = '31' [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.expression.condition.Comparison.mapColumns(Comparison.java:505)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1603)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:832)
2020-12-29 20:18:27 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:16)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:49)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 41 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 35 more
2020-12-29 20:19:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT E.CODEMATI, M.LIBEMATIAR, D.CODEDISC, D.LIBEDISCAR, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM
  EPREUVE AS E
    INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
    INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
    INNER JOIN DISCIPLINE AS D ON M.CODEDISC=D.CODEDISC
WHERE CODEMATI=? AND CODENIVE=? AND CODEPERIEXAM=?; [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.expression.condition.Comparison.mapColumns(Comparison.java:505)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1603)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at tn.manianis.utils.DbConnection.fetchCoefficients(DbConnection.java:46)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2020-12-29 20:20:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT E.CODEMATI, M.LIBEMATIAR, D.CODEDISC, D.LIBEDISCAR, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM
  EPREUVE AS E
    INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
    INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
    INNER JOIN DISCIPLINE AS D ON M.CODEDISC=D.CODEDISC
WHERE CODEMATI=? AND CODENIVE=? AND CODEPERIEXAM=?; [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.expression.condition.Comparison.mapColumns(Comparison.java:505)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1603)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at tn.manianis.utils.DbConnection.fetchCoefficients(DbConnection.java:46)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2020-12-29 20:21:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Ambiguous column name "CODEMATI"; SQL statement:
SELECT E.CODEMATI, M.LIBEMATIAR, D.CODEDISC, D.LIBEDISCAR, CODETYPEMATI, E.CODETYPEEPRE, NUMEEPRE, LIBETYPEEPREAR, ABRETYPEEPREAR, COEFEPRE
FROM
  EPREUVE AS E
    INNER JOIN TYPEEPRE AS TE ON E.CODETYPEEPRE=TE.CODETYPEEPRE
    INNER JOIN MATIERE AS M ON E.CODEMATI=M.CODEMATI
    INNER JOIN DISCIPLINE AS D ON M.CODEDISC=D.CODEDISC
WHERE CODEMATI=? AND CODENIVE=? AND CODEPERIEXAM=?; [90059-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:129)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:107)
	at org.h2.expression.condition.Comparison.mapColumns(Comparison.java:505)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.condition.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1603)
	at org.h2.command.dml.Select.init(Select.java:1125)
	at org.h2.command.Parser.parseQuery(Parser.java:2627)
	at org.h2.command.Parser.parsePrepared(Parser.java:868)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at tn.manianis.utils.DbConnection.fetchCoefficients(DbConnection.java:46)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2020-12-30 05:49:58 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:16)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 41 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 35 more
2020-12-30 05:50:07 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:16)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 41 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 35 more
2020-12-30 06:02:53 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:16)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 06:04:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:16)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:50)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 08:23:31 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 41 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 35 more
2020-12-30 08:23:53 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:316)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 41 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 35 more
2020-12-30 08:24:23 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 08:24:39 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 08:24:55 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 08:25:11 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
2020-12-30 08:25:40 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at tn.manianis.utils.DbConnection.open(DbConnection.java:14)
	at tn.manianis.frames.MainFrame.<init>(MainFrame.java:52)
	at publicationnotes_2.pkg0.PublicationNotes_20$1.run(PublicationNotes_20.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 40 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/manianis/testnotes.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 34 more
